src/test/java/org/generationcp/middleware/service/impl/study/StudyEntryServiceImplIntegrationTest.java

```diff
@@ -54,6 +61,7 @@ public class StudyEntryServiceImplIntegrationTest extends IntegrationTestBase {
 	private static final String ENTRYCODE = "ENTRYCODE-";
 	private static final String CROSS = "ABC/XYZ-";
 	private static final String SEEDSOURCE = "SEEDSOURCE-";
+	private static final String ACCNO = "ACCNO";
 
 	@Autowired
 	private StudyEntryService service;
```

Comment:
@cuenyad could you please rename service to studyEntryService ?

Reply:
done!